### PR TITLE
[Restored] Bump Twilio Library to Version 5.x

### PR DESCRIPTION
**Pull request by acamino**
- [ ] Add missing logo.

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+ruby-2.3.1
```
Comment (smendes) [inline .ruby-version]: Do we need to fix Ruby's version?

File: README.md [hunk 2]
```diff
@@ -2,9 +2,12 @@
```
Comment (smendes) [inline README.md]: We need to add Twilio logo at the very top of the README

File: .ruby-version (hunk 1 above)
Comment (ronualdo) [inline .ruby-version]: I removed the .ruby-version file and added that to .gitignore

File: README.md (hunk 2 above)
Comment (ronualdo) [inline README.md]: Logo added 047033c

**Pull request merged**